terraform/modules/helm/flink/flink.yaml.tfpl

```diff
@@ -48,12 +50,12 @@ base_config: |
   }
 
   postgres {
-    host = postgresql-hl.postgres.svc.cluster.local
+    host = postgresql-hl.postgresql.svc.cluster.local
```

Comment:
We should be using the variable `postgresql_service_name` instead of hard coding the postgres host based on the release name.